src/OpenTelemetry.AutoInstrumentation.StartupBootstrapper/OpenTelemetry.AutoInstrumentation.StartupBootstrapper.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
With the dependency on AspNetCore, should we name this project something like `AspNetCore bootstrapper`? My main concern is having a name that distinguishes itself from the `StartupHook`.

Reply:
Makes sense, how about renaming it to `OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper`?

Reply:
That sounds good to me.